qutebrowser/misc/utilcmds.py

```diff
@@ -217,3 +217,21 @@ def debug_set_fake_clipboard(s=None):
         utils.log_clipboard = True
     else:
         utils.fake_clipboard = s
+
+
+@cmdutils.register()
+@cmdutils.argument('win_id', win_id=True)
+@cmdutils.argument('count', count=True)
+def repeat_command(win_id, count=None):
+    """Repeat the last executed command, like '.' in vi.
+
+    Args:
+        count: Which numeric argument to give the command.
```

Comment:
That sounds odd IMHO, I'd prefer something like "Which count to pass to the command" or so